### PR TITLE
[IMP] website_slides: change confirmation messages for email

### PR DESCRIPTION
**Pull request by fbarbe00**
Purpose
=======
Clean up messages related to email verification.

Specification
=============
Rephrase messages, add confirmation of email sent and let user change
email.

Task-2647065

### CODE REVIEW

File: addons/website_profile/views/website_profile.xml [hunk 1]
```diff
@@ -630,16 +630,22 @@
...
+            <t t-set="my_account_redirect" t-value="'/my/account?redirect='+redirect_url"/>
+            Did not receive it? <a class="alert-link" t-att-href="my_account_redirect"><u>Correct your email address</u></a> 
+            or <a class="send_validation_email alert-link" href="#" t-att-data-redirect_url="redirect_url"><u>Send Again</u></a>.
         </div>
         <div t-if="validation_email_done" t-att-class="done_alert_classes" role="status">
             <button type="button" class="close validated_email_close" data-dismiss="alert" aria-label="Close">×</button>
             <span id="email_validated_message">Congratulations! Your email has just been validated.</span>
             <span t-esc="additional_validated_email_message"/>
         </div>
+
```
Comment (std-odoo) [inline addons/website_profile/views/website_profile.xml:648]: Useless diff

File: addons/website_profile/views/website_profile.xml [hunk 2]
```diff
@@ -283,7 +283,7 @@
                             <div class="o_wprofile_email_validation_container mb16 mt16">
                                 <t t-call="website_profile.email_validation_banner">
                                     <t t-set="redirect_url" t-value="'/profile/user/%s' % user.id"/>
-                                    <t t-set="send_validation_email_message">Click here to send a verification email.</t>
+                                    <t t-set="send_validation_email_message">.</t>
```
Comment (std-odoo) [inline addons/website_profile/views/website_profile.xml:286]: Strange to see a message with just a `.` here, can you just give false here and add the dot (or `!`) in the parent template ?

File: addons/website_profile/views/website_profile.xml [hunk 3]
```diff
@@ -630,16 +630,22 @@
 
         <div t-if="not validation_email_sent and not is_public_user and user.karma == 0" t-att-class="send_alert_classes" role="alert">
             <button type="button" class="close validation_email_close" data-dismiss="alert" aria-label="Close">×</button>
-            It appears your email has not been verified.<br/>
-            <a class="send_validation_email alert-link" href="#" t-att-data-redirect_url="redirect_url">
-                <span t-esc="send_validation_email_message"/>
-            </a>
+            Your Account has not yet been verified.<br/>
+            Click <a class="send_validation_email alert-link" href="#" t-att-data-redirect_url="redirect_url"><u>here</u></a> to receive a verification email<span t-esc="send_validation_email_message"/>
```
Comment (std-odoo) [inline addons/website_profile/views/website_profile.xml:634]: I would add a space before the `<span/>`, also is the span needed ? (no need to create a HTML element, you can use `<t/>`)
```suggestion
            Click <a class="send_validation_email alert-link" href="#" t-att-data-redirect_url="redirect_url"><u>here</u></a> to receive a verification email <t t-esc="send_validation_email_message"/>!
```

File: addons/website_profile/views/website_profile.xml [hunk 4]
```diff
@@ -630,16 +630,22 @@
...
+        </div>
+        <div t-if="validation_email_sent and not validation_email_done" t-att-class="done_alert_classes">
+            <button type="button" class="close" data-dismiss="alert" aria-label="Close">×</button>
+            Verification Email sent to <i><t t-esc="user.email"/></i>!<br/>
```
Comment (std-odoo) [inline addons/website_profile/views/website_profile.xml:638]:
```suggestion
            Verification Email sent to <i t-esc="user.email"></i>!<br/>
```

**Review by std-odoo — COMMENTED** [review]
Hi :)

I added some minor comments, can you have a look ? 

Thanks :)

File: addons/website_profile/views/website_profile.xml (hunk 3 above)
Comment (fbarbe00) [reply addons/website_profile/views/website_profile.xml:634]: I changed the span to t. If no space it added, the sentence will be 
"Click here to receive a verification email !"
with a space before the exclamation mark.

**Review by std-odoo — COMMENTED** [review]
Hi :)

Looking almost good, I added some comments

Thanks for your work

**Review by tde-banana-odoo — COMMENTED** [review]
Blorked a look

**Review by tde-banana-odoo — COMMENTED** [review]
Riviou ! Some final comments :)

**Pull request closed without merging**